data-raw/R2R_TMH_habitat_inputs/documentation/inchannel_rearing_process_and_Review.Rmd

```diff
@@ -0,0 +1,208 @@
+
+[^1]: Stillwater Sciences. 2012. Modeling habitat capacity and population productivity for spring-run Chinook salmon and steelhead in the Upper Yuba River watershed. Technical Report. Prepared by Stillwater Sciences, Berkeley, California for National Marine Fisheries Service, Santa Rosa, California.
+
+FlowWest used the process and data collected by Stillwater Sciences to develop an approach that can be used for all Reorienting to Recovery watersheds that are considered high gradient or valley foothills (outside of the [Habitat Quantification Tool](https://www.sfei.org/sites/default/files/biblio_files/SalmonHQT_UserGuide_2019.pdf) boundary and have a slope greater than 0.4%).
+
+## Objectives
+
+**Objective 1:** Is it possible to use a coarse estimate of gradient by dividing the reach length by the maximum and minimum elevation difference?
+
+**Objective 2:** Calculate the total proportion suitable for North and Middle Yuba Rivers. Use this value and each reaches gradient class to determine an in-channel rearing proportion suitable that can be applied to other reaches within the same gradient class.
+
+```{r include=FALSE}
+# tables from the STillwater Sciences Report 
+table_f_1 <- readxl::read_excel('yuba_inchannel_rearing_tables.xlsx', sheet = "table F-1") |> janitor::clean_names() 
+
+knitr::kable(table_f_1, caption = "Table F-1")
+
+table_f_6 <- readxl::read_excel('yuba_inchannel_rearing_tables.xlsx', sheet = "table F-6") |> janitor::clean_names() 
+
+knitr::kable(table_f_6, caption = "Table F-6")
+
+
+table_4_1 <- readxl::read_excel('yuba_inchannel_rearing_tables.xlsx', sheet = "table 4-1") |> janitor::clean_names() 
+
+knitr::kable(table_4_1, caption = "Table 4-1")
+
+
+```
+
+## Methods
+
+### Objective 1 - coarse vs. fine elevation estimate comparison
+
+As channels steepen, the ratio of suitable salmonid in-channel habitat (pools, riffles, and runs) lessens. The percent of a channel suitable for in-channel rearing depends on how steep the reach is. A coarse elevation profile for each reach was calculated by dividing the elevation difference from top to bottom by the reach length. This is a much simpler approach than calculating the gradient with any finer granularity. A finer approach was used on Middle and North Yuba River to determine if the coarse estimate would be sufficient for this analysis.
```

Comment:
I think you can remove "This is a much simpler approach than calculating the gradient with any finer granularity." makes it stronger and is implied